openfisca_france/parameters/prelevements_sociaux/autres_taxes_participations_assises_salaires/construction/plus_50_salaries.yaml

```diff
@@ -8,6 +8,7 @@ brackets:
       value: 0.0045
```

Comment:
Je ne vois pas l'intérêt d'avoir un bracket basé sur un seuil en PSS ici, il me semble que ce pourrait être une trace d'une tentative de mettre les différents seuils de nombre de salariés dans le même yaml (mais ce n'est de toute évidence pas du tout le cas maintenant). Il serait bon d'essayer de comprendre ce que c'est et au besoin de l'adapter (il s'agit peut-être d'une question de forme pour les barèmes ?).

Reply:
@sylvainipp j'ai regardé en fait ces paramètres ont été mis comme ça pour uniformiser le traîtement dans le preprocessing et le calcul des cotisations dans OF. Je suis d'accord que ce n'est pas très propre mais je propose de laisser ce format.

Reply:
Ce qui m'a étonné était aussi que dans la catégorie moins de 11 salariés ce n'est pas sous la même forme